README.md

```diff
@@ -1,64 +1,169 @@
-# Antispam Bee #
+# Antispam Bee 3.0
```

Comment:
Would not leave a version as static value inside the head, a changelog, verison tag is enough effort.